components/UploadForm/index.js

```diff
@@ -77,7 +87,7 @@ export default function UploadForm({ initialCategories, onSubmit }) {
   return (
     <form onSubmit={handleSubmit}>
       <label htmlFor="name">upload image</label>
-      <input type="text" id="image" name="image" required />
+      <input type="file" id="image" name="imageFile" required />
       <br />
       <label htmlFor="describtion">describtion</label>
```

Comment:
Why break the line? You can do that with styling your form :)
also a typo in description

Reply:
Oh f***, I think I wrote 'description' everywhere wrong. I'll try to fix it.

Reply:
The breaks I wanna change, when I start styling.